testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
OK, I've read the page you've pointed to, and I've read your explanation.  Your implementation matches their description.  The feature is just screwy and unintuitive.

BTW, public vs. private on the accessor functions is /overridden/.  The private keyword has become meaningless, and it's more likely to create bugs than not.  Particularly since we have code that checks for the existence of a private keyword to determine whether something is private.  We're going to have to find and fix those now, too.